models/SymbolMaster/SymbolMaster.test.js

```diff
@@ -63,4 +67,239 @@ describe('SymbolMaster', () => {
     expect(symbolInstance.symbolID).toEqual(symbolMaster.symbolID);
     expect(symbolInstance.overrideValues[0].overrideName).toEqual(symbolMaster.overrideProperties[0].overrideName);
   });
+
+  it('should be able to create a symbol instance', () => {
+    const symbolMaster = new SymbolMaster({
+      name: 'symbol',
+    });
+
+    const layer = new Text({
+      name: 'text',
+    });
+
+    const instanceName = 'test instance';
+
+    symbolMaster.addLayer(layer, true);
+    const symbolInstance = symbolMaster.createInstance({ name: instanceName });
+
+    expect(symbolInstance.symbolID).toEqual(symbolMaster.symbolID);
+    expect(symbolInstance.name).toEqual(instanceName);
+    expect(symbolInstance.overrideValues[0].overrideName).toEqual(symbolMaster.overrideProperties[0].overrideName);
+  });
+
+  it('should be able to create a symbol instance with text overrides', () => {
+    const symbolMaster = new SymbolMaster({
+      name: 'symbol',
+    });
+
+    const layer = new Text({
+      name: 'text',
+    });
+
+    const overrideText = 'override text';
+
+    symbolMaster.addLayer(layer, true);
+    const symbolInstance = symbolMaster.createInstance({
+      overrideValues: [
+        {
+          name: 'text',
+          value: overrideText,
+        },
+      ],
+    });
+
+    expect(symbolInstance.overrideValues[0].value).toEqual(overrideText);
+  });
+
+  it('should be able to create a symbol instance with layer style overrides', () => {
+    const beforeStyle = new SharedStyle({
+      name: 'before',
+      fills: [
+        {
+          color: '#fff',
+        },
+      ],
+    });
+
+    const afterStyle = new SharedStyle({
+      name: 'after',
+      fills: [
+        {
+          color: '#000',
+        },
+      ],
+    });
+
+    const symbolMaster = new SymbolMaster({
+      name: 'symbol',
+    });
+
+    const layer = new Rectangle({
+      width: 200,
+      height: 100,
+      x: 0,
+      y: 0,
+      name: 'rectangle',
+      style: beforeStyle,
+    });
+
+    symbolMaster.addLayer(layer, true);
+
+    // symbolMaster.addLayer currenty sets the overrideName incorrectly - temp fix
```

Comment:
Could we fix this? Or maybe we do this in another PR

Reply:
committed the changes for this